### DISCUSSION TITLE
Tag para el primer commit del grupo en la Prueba de Concepto

### DISCUSSION THREAD

**Discussion by NicoCollazo** [dds-jv/foro#116] [Consultas de herramientas] [answered]
Hola!
La duda que tenemos es con respecto a el siguiente bulletpoint de la Prueba de Concepto:

> Se debe generar un tag (o concepto similar) en el repositorio, indicando que esa es una versión estable de la entrega.

Ese tag, ¿debe ser un git tag, o debe ser un badge como el que te genera Travis que indique que los tests se ejecutan exitosamente?

Gracias y saludos

**Comment by gastonprieto** [comment]
Hola!

No, el badge es otra cosa, es algo que te muestra el estado. Lo que pedimos es un tag de git que le da una etiqueta a un commit en particular.

**Comment by gastonprieto** [comment]
@NicoCollazo te sirvio?

**Comment by NicoCollazo** [comment, marked as answer]
Sì! Ya lo pudimos solucionar. Ahí cierro el issue
